tests/mocks/mock_embedding_driver.py

```diff
@@ -12,6 +14,11 @@ class MockEmbeddingDriver(BaseEmbeddingDriver):
     dimensions: int = field(default=42, kw_only=True)
     max_attempts: int = field(default=1, kw_only=True)
     tokenizer: MockTokenizer = field(factory=lambda: MockTokenizer(model="foo bar"), kw_only=True)
+    mock_output_function: Callable[[str], Optional[list[float]]] = field(default=lambda chunk: None, kw_only=None)
```

Comment:
Let's rename to `mock_output` [for consistency](https://github.com/griptape-ai/griptape/blob/6b7e8e9989f022a0743ad8b3b02d8afb348f014e/tests/mocks/mock_prompt_driver.py#L33) with `MockPromptDriver` and have the default be `Factory(lambda chunk: [0, 1])`.

Then `try_embed_chunk` can just become:
```python
    def try_embed_chunk(self, chunk: str) -> list[float]:
        return self.mock_output_function(chunk)
```